ocaml-lsp-server/src/document.ml

```diff
@@ -13,7 +13,9 @@ module Kind = struct
     | ".mli"
     | ".rei" ->
       Intf
-    | ext -> failwith ("Unknown extension " ^ ext)
+    | ext ->
+      Code_error.raise "unsupported file extension"
```

Comment:
Is it possible for the user to trigger this error?

Reply:
The input argument is a `Document.t`, and from what I can see the only way to get that is to get the doc from the store. So a user should not be able to trigger this unless there's a bug.

Reply:
That's not true, I wasn't looking at the right place.